src/components/Patient/PatientDetailsTab/HealthProfileSummary.tsx

```
@@ -0,0 +1,125 @@
+
+  const handleEditClick = (sectionId: any) => {
+    navigate(
+      `/facility/${facilityId}/patient/${id}/update?section=${sectionId}`,
+    );
+  };
+
+  let patientMedHis: any[] = [];
+  if (
+    patientData &&
+    patientData.medical_history &&
+    patientData.medical_history.length
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)